cibyl/cli/main.py

```diff
@@ -130,5 +121,21 @@ def main():
         verbosity=orchestrator.parser.app_args.get('verbosity'))
 
 
+def main():
+    """CLI main entry."""
+    debug = False
+
+    if '--debug' in sys.argv:
```

Comment:
I have moved the try...except into the 'run_cibyl' function so that I can reuse the argument parser. The only thing that is not handled right now is the step where arguments are read, but it makes sense, as I need to read the arguments before knowing if I am in debug mode or not.